pallets/remarks/src/benchmarking.rs

```diff
@@ -29,7 +29,7 @@ mod benchmarks {
 	use super::*;
 
 	#[benchmark]
-	fn remark() -> Result<(), BenchmarkError> {
+	fn remark(n: Linear<1, 10>) -> Result<(), BenchmarkError> {
```

Comment:
Why 10? Just curoius about the new syntax.

Reply:
Ideally, I would've used `T::MaxRemarksPerCall::get()`, similar to how I did it in the initial implementation, but that is no longer possible. So I used 10 since it's the max number of remarks we'll use in centchain.

Reply:
Perfect! Can you create a doc comment for that?

Reply:
Done.